spec/system/adjustment_system_spec.rb

```diff
@@ -54,6 +54,22 @@
         expect(page).to have_no_content(item.name)
       end
     end
+
+    it "politely informs the user that they're adjusting way too hard", js: true do
```

Comment:
that's the only way I like to adjust things... much like a bear adjusting a band on a shed